### ISSUE TITLE
Platform: All field labels should have consistent capitalisation

### ISSUE THREAD

**Issue by matt-west** [Automattic/wp-calypso#100240]
## Where

We use inconsistent capitalisation for form fields throughout the product.

An example from /me:
<img width="197" alt="Image" src="https://github.com/user-attachments/assets/547be4bb-b5f5-4e7c-8c9c-280746dabed0" />

##  Solution

Adopting form field components from the @wordpress/components should standardise form labels so that they are all consistent.

<img width="238" alt="Image" src="https://github.com/user-attachments/assets/a2186a1e-046d-43c3-8e73-4e03d127b2d5" />

**Comment by crisbusquets** [comment]
Based on the discussion here (p58i-ks2-p2) and @michaelpick's comment (p58i-ks2-p2#comment-67169), we should update our copy to **Sentence Case**.

Updated this to `Ready for Dev`.

**Comment by jasmussen** [comment]
> Adopting form field components from the @wordpress/components should standardise form labels so that they are all consistent.

Adding some nuance: the labels can still be uppercase for labels, through `text-transform: uppercase;` yes? Agreed the actual copy, across the board, should be sentence case as written. Just noting, as the uppercase treatment is the default for the core componentry, and the block editor, we should keep that.
